src/main/java/com/gettyimages/api/Credentials.java

```diff
@@ -74,12 +74,12 @@ public static Credentials GetInstance(String apiKey, String apiSecret, String us
     }
 
     public Token GetAccessToken() throws SdkException {
-        Calendar now = Calendar.getInstance();
-        now.add(Calendar.MINUTE, 5);
+        Calendar expirationCushion = Calendar.getInstance();
```

Comment:
I didn't want to use "buffer" although it's probably a better word for what this variable is, at least outside of programming.